src/create.rs

```diff
@@ -273,14 +270,14 @@ pub struct Party2 {
     TX_f: FundingTransaction,
     TX_c: CommitTransaction,
     TX_s: SplitTransaction,
-    sig_TX_c_self: AdaptorSignature,
-    sig_TX_s_self: secp256k1::Signature,
+    presig_TX_c_self: EncryptedSignature,
```

Comment:
mixing terminology: "presig" and "encrypted  signature".

Reply:
Yep. Caught in between two worlds. I was thinking of introducing our own signature and presignature types so that we can own the types for our model and avoid confusions.